discovery.md

```diff
@@ -109,296 +123,217 @@ to execute some logic, which might lead to the occurrence of new events.
 
-The request for events from an Event Producer system.
+The request for events from a Service.
+
 
 ## API Specification
 
-This API is specified as a REST API with well defined entities and relationships
-between those entities.
+This API is specified as a REST API with well defined entities and
+relationships between those entities.
+
+### Services
 
-To help explain the data model, the following non-normative pseudo yaml shows
-its basic conceptual structure:
+At the core of the data model is the concept of a [Service](#service). The
+API then exposes multiple ways to query over a list of Services. To help
+explain the Service resource, the following non-normative pseudo json shows
+its basic structure:
 
-( `*` means zero or more, `+` means one or more, `?` means optional)
+(`*` means zero or more, `+` means one or more, `?` means optional)
 
+Service:
 ```
-- services: *
-  - service: [unique identifier for producer of the specified event types]
-    description: [human string] ?
-    uri: [URI reference for human documentation] ?
-    specversion: [ce-speciversion value] ? (required?)
-    subscriptionuri: [URI to which the Subscribe request will be sent]
-    subscriptionconfig: *
-    authscope: [string] ?  (explain)
-    protocols: +
-    - protocol: [string]
-    types: *
-    - type: [ce-type value]
-      description: [human string] ?
-      datacontenttype: [ce-datacontenttype value] ?
-      dataschema: [ce-dataschema URI] ?
-      dataschematype: [string per RFC 2046] ?
-      dataschemacontent: [schema] ?
-      dataschemacontent:
-      extensions: *
-      - name: [CE context attribute name]
-        type: [CE type string]
-        spec: [URI to specification defining the extension] ?
+{
+  "url": "[unique URL to this service]",
+  "description": "[human string]", ?
+  "docsurl": "[URL reference for human documentation]", ?
+  "specversion": "[ce-specversion value]",
+  "subscriptionurl": "[URL to which the Subscribe request will be sent]",
+  "subscriptionconfig": { ?
+    "[key]": "[value]", *
+  },
+  "authscope": "[string]", ?
+  "protocols": [ "[string]" + ],
+  "types": [ ?
+    { *
+      "type": "[ce-type value]",
+      "description": "[human string]", ?
+      "datacontenttype": "[ce-datacontenttype value]", ?
+      "dataschema": "[ce-dataschema URI]", ?
+      "dataschematype": "[string per RFC 2046]", ?
+      "dataschemacontent": "[schema]", ?
+      "extensions": [ ?
+        { *
+          "name": "[CE context attribute name]",
+          "type": "[CE type string]",
+          "specurl": "[URL to specification defining the extension]" ?
+        }
+      ]
+    }
+  ]
+}
 ```
 
 An example:
-```
-- services:
-  - service: example.com
-    subscriptionuri: https://events.example.com
-    protocols:
-    - protocol: HTTP
-    types:
-    - type: com.example.widget.create
-    - type: com.example.widget.delete
+```json
+{
+  "url": "https://example.com/services/widgetService",
+  "specversion": "v1.0",
```

Comment:
`specversion` should be `"1.0"` not `"v1.0"`

Reply:
fixed